pycode/epidemiology/epidata/getDIVIData.py

```diff
@@ -659,15 +224,42 @@ def get_divi_data(read_data=dd.defaultDict['read_data'],
     df_ger = df.groupby(["Date"]).agg({"ICU": sum, "ICU_ventilated": sum})
     df_ger = df_ger.reset_index()
     df_ger.sort_index(axis=1, inplace=True)
-    # for performing sum nan is treated like an zero
-    # However, afterwards ist should be filled with NaN
-    # TODO: should we have this? See issue #205
-    # df_ger.loc[df_states["Date"] <= "2020-04-25 09:15:00", "ICU_ventilated"] = np.nan
-    # TODO: Use also "faelle_covid_aktuell_im_bundesland" from 25.9.
+
     filename = "germany_divi"
     filename = gd.append_filename(filename, impute_dates, moving_average)
     gd.write_dataframe(df_ger, directory, filename, file_format)
 
+    return(df_raw, df_counties, df_states, df_ger)
+
+
+def gdd_sanity_checks(df = pd.DataFrame()):
```

Comment:
This functions, as other new functions, does not have any documentation! Please add it.